File: server/BidMasterOnline/BidMasterOnline.Domain/Models/Entities/UserFeedback.cs
namespace BidMasterOnline.Domain.Models.Entities
{
    public class UserFeedback : EntityBase
    {
        public long FromUserId { get; set; }

        public long ToUserId { get; set; }

        public int Score { get; set; }

        public string? Comment { get; set; }
    }
}

File: client/ModeratorUI/src/app/models/users/userFeedback.ts
import { BaseEntityModel } from "../baseEntityModel";

export class UserFeedback extends BaseEntityModel {
  public fromUserId!: number;
  public toUserId!: number;
  public score!: number;
  public comment!: string;
  public fromUsername!: string;
  public toUsername!: string;
}
